docs/api/ShallowWrapper/prop.md

```diff
@@ -1,22 +1,44 @@
 # `.prop(key) => Any`
 
-Returns the prop value for the node of the current wrapper with the provided key.
+Returns the prop value for the root node of the wrapper with the provided key.
+`.prop(key)` can only be called on a wrapper of a single node.
 
-NOTE: can only be called on a wrapper of a single node.
+NOTE: When called on a shallow wrapper, `.prop(key)` will only return values for props that exist for the parent element of the component, not the whole React component. To return the props for the
+entire React component, use `wrapper.instance().props`. See [`.instance() =>
+ReactComponent`](instance.md)
 
 #### Arguments
 
 1. `key` (`String`): The prop name such that this will return value will be the `this.props[key]`
-of the component instance.
+of the parent element of the component.
```

Comment:
similarly, "parent element" isn't really the right verbage here - it's "of the root node that the component *renders*"